crates/polars-time/src/windows/test.rs

```diff
@@ -148,8 +148,8 @@ fn test_groups_large_interval() {
         false,
         Default::default(),
     );
-    assert_eq!(groups.len(), 2);
-    assert_eq!(groups[1], [2, 2]);
+    assert_eq!(groups.len(), 3);
+    assert_eq!(groups[1], [1, 1]);
```

Comment:
same story, earliest datapoint not included in any group due to `closed='right'`